docs/design/add-in-icons-monoline.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Monoline style icon guidelines for Office Add-ins
+description: ''
+ms.date: 12/09/2019
+localization_priority: Priority
+---
+
+# Monoline style icon guidelines for Office Add-ins
+
+Monoline style iconography are used in Office 365. If you would prefer that your icons match the Fresh style of non-subscription Office 2013+, see [Fresh style icon guidelines for Office Add-ins](add-in-icons-fresh.md).
+
+## Office Monoline visual style
+
+The goal of the Monoline style to have consistent, clear, and accessible iconography to communicate action and features with simple visuals, ensure the icons are accessible to all users, and have a style that is consistent with those used elsewhere in Windows.
+
+The following guidelines are for 3rd party developers who want to create icons for features that will be consistent with the icons already present Office products.
+
+### Design principles
+
+-   Simple, clean, clear.
+-   Contain only necessary elements.
+-   Inspired by Windows icon style.
+-   Accessible to all users.
+
+#### Conveying meaning
+
+-   Use descriptive elements such as a page to represent a document or an envelope to represent mail.
+-   Use the same element to represent the same concept, i.e. mail is always represented by an envelope, not a stamp.
+-   Use a core metaphor during concept development.
+
+#### Reduction of Elements
+
+-   Reduce the icon to its core meaning, using only elements that are essential to the metaphor.
+-   Limit the number of elements in an icon to two, regardless of icon size.
+
+#### Consistency
+
+Sizes, arrangement, and color of icons should be consistent.
+
+#### Styling
+
+##### Perspective
+
+Monoline icons are forward-facing by default. Certain elements that require perspective and/or rotation, such as a cube are allowed, but exceptions should be kept to a minimum.
+
+##### Embellishment
+
+Monoline is a clean minimal style. Everything uses flat color, which means there are no gradients, textures, or light sources.
+
+## Designing
+
+### Sizes
+
+We recommend that you produce each icon in all these sizes to support high DPI devices. The absolutely *required* sizes are 16x, 20x, and 32x, as those are the 100% sizes.
```

Comment:
Should all these sizes throughout the article use `px` instead of `x` (e.g. 16px instead of 16x)?

Reply:
The "x" by itself is a way of expressing the size that designers understand.

Reply:
Granted I'm not a designer but because this is in Sizes section I initially thought they were magnifications.

Reply:
@ElizabethSamuel-MSFT After thinking it over last night, I will change this to "px".